src/System.Net.Mail/src/System/Net/Mail/SmtpConnection.Auth.cs

```diff
@@ -16,7 +16,6 @@ internal partial class SmtpConnection
     {
         private bool _serverSupportsEai;
         private bool _dsnEnabled;
-        private bool _serverSupportsStartTls;
```

Comment:
Why did you remove this?

Reply:
Per #17905 and http://tempcoverage.blob.core.windows.net/report3/System.Net.Mail.diff.html, `internal bool ServerSupportsStartTls { get; }` was to be removed as dead code.

That led to a build error:
> D:\Repos\corefx\src\System.Net.Mail\src\System\Net\Mail\SmtpConnection.Auth.cs(19,22): error CS0414: The field 'SmtpConnection._serverSupportsStartTls' is assigned but its value is never used [D:\Re
pos\corefx\src\System.Net.Mail\tests\Unit\System.Net.Mail.Unit.Tests.csproj]

I saw "assigned but its value is never used" and immediately thought I could delete it. I didn't notice that this error actually came up only for the test project.

Fixing...